config/application.rb

```diff
@@ -68,10 +68,6 @@ class Application < Rails::Application
     # Rails 5.0 had false.
     config.active_record.belongs_to_required_by_default = false
 
-    # Temporary workaround for:
-    # https://discuss.rubyonrails.org/t/cve-2022-32224-possible-rce-escalation-bug-with-serialized-columns-in-active-record/81017
```

Comment:
This doesn't look to be part of the ticket. Are we just good to remove from our earlier discussion this morning?

Reply:
Yes. Removing this config since it's not doing anything anymore.

Reply:
Actually it still exists in rails. I'm going to put it back for now.